crates/pop-fork/src/models.rs

```diff
@@ -0,0 +1,20 @@
+use crate::schema::{blocks, storage};
+use diesel::Insertable;
+
+#[derive(Insertable, Clone)]
+#[diesel(table_name = storage)]
+pub(crate) struct NewStorageRow {
+	pub(crate) block_hash: Vec<u8>,
+	pub(crate) key: Vec<u8>,
+	pub(crate) value: Option<Vec<u8>>,
+	pub(crate) is_empty: bool,
+}
+
+#[derive(Insertable, Clone)]
+#[diesel(table_name = blocks)]
+pub(crate) struct NewBlockRow {
+	pub(crate) hash: Vec<u8>,
+	pub(crate) number: i32,
```

Comment:
As @tsenovilla hinted previously we might want to use i64 instead.

Polkadot SDK blocks are u32, but we will always fit them in i64 and have safe conversions.
After the previous feedback I went with a `try_into`, but it might be worth reconsidering.

Reply:
Let's do it :) Thanks